Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -106,6 +106,21 @@ public enum eBrowserType
         [UserConfiguredDescription("Use Browser In Private/Incognito Mode (Please use 64bit Browse with Internet Explorer ")]
         public bool BrowserPrivateMode { get; set; }
 
+        [UserConfigured]
+        [UserConfiguredDefault("false")]
+        [UserConfiguredDescription("Only for Chrome & Firefox | Set \"true\" to run the browser in background (headless mode) for faster Execution")]
+        public bool HeadlessBrowserMode { get; set; }
+
+        [UserConfigured]
+        [UserConfiguredDefault("false")]
+        [UserConfiguredDescription("Only for IE, Chrome & Firefox | Set \"true\" to Launch the Browser minimized")]
+        public bool BrowserMinimized { get; set; }
+
+        [UserConfigured]
+        [UserConfiguredDefault("false")]
```

Comment:
defualt value for HideConsoleWindow should be "true" we don't want the console as defualt

Reply:
i see defualt value is still "false"